lib/ClangImporter/ClangModuleDependencyScanner.cpp

```diff
@@ -254,6 +254,20 @@ void ClangImporter::recordModuleDependencies(
     swiftArgs.push_back("-module-name");
     swiftArgs.push_back(clangModuleDep.ModuleName);
 
+    // Pass down search paths to the -emit-module action.
+    // Unlike building Swift modules, we need to include all search paths to
+    // the clang invocation to build PCMs because transitive headers can only
+    // be found via search paths. Passing these headers as explicit inputs can
+    // be quite challenging.
+    for (auto &path: Impl.SwiftContext.SearchPathOpts.ImportSearchPaths) {
+      swiftArgs.push_back("-I");
```

Comment:
Do we not need `addClangArg` here so that the swift invocation that uses these commands passes them down to the underlying clang invocation?

Reply:
We shouldn't need them because search paths are added directly to a clang instance via `ClangImporter::addSearchPath`.

Reply:
Thanks for the clarification! 👍